File: 1043.c
#include <stdio.h>

int main(){

    float a, b, c, perimetro, area;

    scanf("%f %f %f", &a, &b, &c);

    if((a + b > c && c > a + b - c) || (a + c > b && b > a - c) || (b + c > a && a > b - c)){

        perimetro = a + b + c;

        printf("Perimetro = %.1f\n", perimetro);
    }else{

        area = ((a + b) / 2) * c;
        printf("Area = %.1f\n", area);
    }


    return 0;

}

File: Iniciante/1043.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner scanf = new Scanner(System.in);

        float perimetro, area;
        float a = scanf.nextFloat();
        float b = scanf.nextFloat();
        float c = scanf.nextFloat();

        if((a + b > c && c > a + b - c) || (a + c > b && b > a - c) || (b + c > a && a > b - c)){

            perimetro = a + b + c;
    
            System.out.printf("Perimetro = %.1f\n", perimetro);
        }else{
    
            area = ((a + b) / 2) * c;
            System.out.printf("Area = %.1f\n", area);
        }
    }
}